src/services/streaming-manager/index.ts

```diff
@@ -292,6 +291,22 @@ export async function createStreamingManager<T extends CreateStreamOptions>(
                 clearInterval(videoStatsInterval);
             }
         },
+        /**
+         * Method to send data channel messages to the server
+         */
+        sendDataChannelMessage(payload: string) {
+            if (!isConnected || pcDataChannel.readyState !== 'open') {
+                log('Data channel is not ready for sending messages');
+                return;
```

Comment:
I think we should indicate this to the caller. This looks like a good place to throw an error (and catch is someplace else)